### DISCUSSION TITLE
Support for database URLs

### DISCUSSION THREAD

**Discussion by mikerockett** [spatie/db-dumper#195] [Ideas]
My app is being deployed to Fly.io, which uses `DATABASE_URL` for connections. Whilst one could easily populate the applicable environment variables, like `DB_HOST`, it would be nice to see this package support `DATABASE_URL`, which Laravel supports out of the box (in which case the laravel-db-snapshots package would need to be updated to support this).

**Comment by freekmurze** [comment]
If the code is simple, I'd accept a PR that adds this. Make sure to update docs and tests.

**Reply by mikerockett** [reply]
I’ve opened #196 for your review. Tests were failing, all passing now.
